src_rebuild/GAME/C/PRES.C

```diff
@@ -3,7 +3,7 @@
 #include "SYSTEM.H"
 #include "TEXTURE.H"
 
-#include <STRING.H>
+#include <string.h>
```

Comment:
This is not correct as it's included from PSY-Q. I'm expecting to backport it to PSX

Reply:
I changed this because `STRING.H` was not found. On Windows this include may be automatically resolved to `string.h`, since paths are case-insensitive (unlike on Linux). Should we change this to `STRINGS.H` instead?

Reply:
yep, correct. There should be STRINGS.H